src/app/catalogue/asset/components/Recommender.jsx

```diff
@@ -0,0 +1,24 @@
+import Link from 'next/link'
+import OfferingItem from '../../components/OfferingItem'
+function Recommender ({ recommendations }) {
+  return (
+    <div className='bg-white w-3/3 m-5 mr-10 p-5 rounded-md'>
+      <h2 className='font-bold text-2xl'>You may also like...</h2>
+      <div className='flex gap-4 overflow-x-auto p-4 w-full'>
+        {recommendations.map((offering, index) => {
+          return (
+            <Link key={`${offering.title}-${offering.created_at}-${index + 1}`} href='#'>
+              <OfferingItem
```

Comment:
It looks a bit cramped with `OfferingItem` out of the box, and in case the name of the provider ends up being a little long, it may look ugly. Anyway, since we do not have the backend yet for all this, I would suggest not spending time on this, we can simply remove the _dataset_ entry, and impose a minimum space between the left and right side of the card. What do you think ?

Reply:
I created a new component with a few modifications, let me know what you think :)